packages/elements-react-example/src/App.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { BrowserRouter as Router, Navigate, Route, Routes } from 'react-router-dom';
 import './index.scss';
-import logo from './favicon.svg';
+import logo from '../../../assets/logo/elements.svg';
```

Comment:
Should we define a `paths`  value for this to access these files like `@assets/logo/elements.svg` or `@logo/elements.svg`

Reply:
I tried to extend the root `tsconfig` with the path mappings but vite does not seem to find the files then 🤔

Reply:
Okay, this could relate to the missing extension of the base `tsconfig` in the packages. But we can this optimize later too :)